packages/office-ui-fabric-react/src/components/Panel/Panel.base.tsx

```diff
@@ -295,28 +297,39 @@ export class PanelBase extends BaseComponent<IPanelProps, IPanelState> implement
       // const iconButtonStyles = this._classNames.subComponentStyles
       // ? (this._classNames.subComponentStyles.iconButton as IStyleFunctionOrObject<IButtonStyleProps, IButtonStyles>)
       // : undefined;
+      const headerTextId = this.props.headerText && this.state.id + '-headerText';
+      const header = this._onRenderHeader(this.props, this._onRenderHeader, headerTextId);
+      const isHeaderInContentInner = !!(this.props.onRenderNavigation || this.props.onRenderNavigationContent);
```

Comment:
can't we use the global one of these?

Reply:
added `_hasCustomNavigation` as a private member variable whose value is set by the expression you highlight above.